.github/home.css

```
@@ -0,0 +1,495 @@
+    background-clip: text;
+    -webkit-background-clip: text;
+    color: transparent;
+}
+
+/* signup button */
+.section1 > a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    width: 350px;
+    height: 54px;
+    padding: 16px 20px;
+    border-radius: 8px;
+    border: none;
+    background: linear-gradient(91deg, #6D6AFE 0.12%, #6AE3FE 101.84%);
+    color: #f5f5f5;
+    font-family: Pretendard;
+    font-size: 18px;
+    font-weight: 600;
+    text-decoration: none;
+}
+
+.section1 > a:hover {
+    cursor: pointer;
+}
+
+
+/*section1 image*/
+.section1_img {
+    width: 1200px;
+    height: 590px;
+    overflow: hidden;
+    padding-top: 50px;
+}
+
+.section1_img > img {
+    width: 1118px;
+    flex-shrink: 0;
+    border-radius: 25px;
+    background: linear-gradient(180deg, rgba(196, 196, 196, 0.00) 67.68%, #f0f6ff 94.76%);
+    box-shadow: 0px 4px 25px 0px rgba(0, 0, 0, 0.00);
+    object-fit: cover;
+}
+
+
+/*section2*/
+.section2 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 120px 461px 50px 461px;
+    width: 1920px;
+    height: 620px;
+    background-color: #ffffff;
+}
+
+.section2_contents {
+    width: 998px;
+    height: 450px;
+    display: flex;
+    align-items: center;
+    gap: 157px;
+    flex-direction: row;
+}
+
+.section2_text {
+    display: flex;
+    width: 315px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 10px;
+}
+
+.section2_text > h2 {
+    font-family: Pretendard;
+    font-size: 48px;
+    font-weight: 700;
+    letter-spacing: -0.3px;
+    color: #000000;
+    margin: 0;
+}
+
+.section2_text > h2 > span {
+    background-image: linear-gradient(96deg, #FE8A8A 1.72%, #A4CEFF 74.97%);
+    background-clip: text;
+    -webkit-background-clip: text;
+    color: transparent;
+}
+
+.section2_text > p {
+    color: #6b6b6b;
+    font-family: Pretendard;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 150%;
+    margin: 0;
+    width: 298px;
+}
+
+.section2_card {
+    width: 550px;
+    height: 450px;
+    border-radius: 15px;
+    background: #f0f6ff;
+    display: inline-flex;
+    align-items: center;
+    gap: -12px;
+    justify-content: center;
+    overflow: hidden;
+}
+
+.section2_card > img {
+    width: 279px;
+    height: 274.927px;
+}
+
+.section2_card .blueimg {
+    position: relative;
+}
+
+.section2_card .icon {
+    width: 46px;
+    height: 46px;
+    flex-shrink: 0;
+    position: absolute;
+    left: 1290px;
+    top: 1500px;
+}
+
+.section3 {
+    padding: 50px 461px;
+    background-color: #ffffff;
+    width: 1920px;
+    height: 550px;
+}
+
+.section3_contents {
+    gap: 157px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    gap: 157px;
+    width: 998px;
+    height: 450px;
+}
+
+.section3_img {
+    width: 550px;
+    padding: 114px 68px 66px 68px;
```

```suggestion
    padding: 114px 68px 66px;
```
사소한 부분이긴 한데 위, 아래 영역을 달리 정하고 좌,우 영역이 같다면
위와 같이 수정할 수 있겠습니다.